airbyte-config/models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -12,125 +12,305 @@
 import java.util.Map;
 import java.util.Set;
 
+/**
+ * This interface defines the general variables for configuring Airbyte.
+ *
+ * Please also update the configuring-airbyte.md document when modifying this file.
+ */
 public interface Configs {
 
   // CORE
   // General
+  /**
+   * Distinguishes internal Airbyte deployments. Internal-use only.
+   */
   String getAirbyteRole();
 
+  /**
+   * Defines the Airbyte deployment version.
+   */
   AirbyteVersion getAirbyteVersion();
 
   String getAirbyteVersionOrWarning();
 
+  /**
+   * Defines the bucket for caching specs. This immensely speeds up spec operations. This is updated
+   * when new versions are published.
+   */
   String getSpecCacheBucket();
 
+  /**
+   * Distinguishes internal Airbyte deployments. Internal-use only.
+   */
   DeploymentMode getDeploymentMode();
 
+  /**
+   * Defines if the deployment is Docker or Kubernetes. Airbyte behaves accordingly.
+   */
   WorkerEnvironment getWorkerEnvironment();
 
+  /**
+   * Defines the configs directory. Applies only to Docker, and is present in Kubernetes for backward
+   * compatibility.
+   */
   Path getConfigRoot();
 
+  /**
+   * Defines the Airbyte workspace directory. Applies only to Docker, and is present in Kubernetes for
+   * backward compatibility.
+   */
   Path getWorkspaceRoot();
 
   // Docker Only
+  /**
+   * Defines the name of the Airbyte docker volume.
+   */
   String getWorkspaceDockerMount();
 
   String getLocalDockerMount();
```

Comment:
@cgardens @jrhizor can you help fill this? I can do word smithing after.

Reply:
The name of the docker mount that is used for local file handling. On Docker, this allows connector pods to interact with a volume for "local file" operations.

Reply:
Thanks!